cmd/template-resolver/utils/templateResolver.go

```diff
@@ -18,6 +18,26 @@ import (
 	"github.com/stolostron/go-template-utils/v6/pkg/templates"
 )
 
+type hubTemplateCtx struct {
+	ManagedClusterName   string
+	ManagedClusterLabels map[string]string
+}
+
+type hubTemplateOptions struct {
+	config templates.Config
+	opts   templates.ResolveOptions
+	ctx    hubTemplateCtx
+}
+
+var defaultHubTemplateOpts = &hubTemplateOptions{
+	config: templates.Config{
+		AdditionalIndentation: 8,
+		DisabledFunctions:     []string{},
+		StartDelim:            "{{hub",
+		StopDelim:             "hub}}",
+	},
+}
```

Comment:
There's not a benefit to making this a package-level variable--it should be function-scoped.